architecture/job.go

```diff
@@ -35,13 +35,14 @@ type Job struct {
 }
 
 func NewJob(id string, pri, delay, ttr, bytes int64, data string) *Job {
-	j := new(Job)
-	j.id = id
-	j.Pri = pri
-	j.Delay = delay
-	j.TTR = ttr
-	j.Bytes = bytes
-	j.Data = data
+	j := &Job{
```

Comment:
I know this is my mistake, but I would like to see this variable renamed to `job` instead of just a `j` if you could do so.